source/ADIOS.cpp

```diff
@@ -25,10 +25,8 @@
 #include "engine/bp/BPFileReader.h"
 #include "engine/bp/BPFileWriter.h"
 
-#ifdef ADIOS_HAVE_DATAMAN // external dependencies
 #include "engine/dataman/DataManReader.h"
```

Comment:
This should still be guarded with an `#ifdef` since the DataMan code will only be supported for shared lib builds.